CMakeLists.txt

```diff
@@ -54,9 +56,26 @@ add_subdirectory(input)
 add_subdirectory(src)
 
 if (ISO15118_BUILD_TESTING)
+    set(CMAKE_BUILD_TYPE Debug CACHE STRING "Build type" FORCE) # TODO(sl): Check if necessary
+
     include(CTest)
     list(APPEND CMAKE_MODULE_PATH ${CPM_PACKAGE_catch2_SOURCE_DIR}/extras)
     add_subdirectory(test)
+
+    # TODO(sl): Maybe move to test folder
+    set(CODE_COVERAGE_VERBOSE ON)
+    set(GCOVR_ADDITIONAL_ARGS "--gcov-ignore-errors=no_working_dir_found")
```

Comment:
is this with gcovr 8 still necessary?

Reply:
Without this the CI jobs does not work anymore. So I added this again for the CI